src/sentry/utils/sdk_crashes/event_stripper.py

```diff
@@ -49,6 +53,16 @@ def _filter_contexts(self, pair):
             return True
         return False
 
+    def _strip_debugMeta(
```

Comment:
https://peps.python.org/pep-0008/#function-and-variable-names

```suggestion
    def _strip_debug_meta(
```